ApplicationCode/Application/RiaSummaryCurveDefinition.h

```diff
@@ -58,6 +60,7 @@ class RiaSummaryCurveDefinition
     RimSummaryCase*           m_summaryCase;
     RifEclipseSummaryAddress  m_summaryAddress;
     RimSummaryCaseCollection* m_ensemble;
+    bool                      m_treatAsSingleSummaryCurve;
```

Comment:
Maybe change to m_isStandaloneSummaryCurve?

Reply:
changing it to be opposite and "m_isEnsembleCurve".